composer.json

```diff
@@ -37,7 +37,7 @@
         "ext-iconv": "Can be used as fallback when ext-mbstring is not available"
     },
     "require-dev": {
-        "bamarni/composer-bin-plugin": "^1.4",
```

Comment:
newer phpstan versions no longer define external dependencies. we don't need the composer bin plugin anymore

Reply:
The main issue is that the 1.x branch supports PHP versions that are not supported by phpstan.

Reply:
ohh wow.. PHP5 is kind of old, didn't realize. thanks for the hint.

is it fine for the master branch?

Reply:
For now, I'd rather keep the same setup in both branches. It makes it simpler to merge up.

Once the 2.0 rewrite is over and I can stop maintaining 1.x, I will reconsider